RCM-DX-Specification_EN.md

```diff
@@ -324,7 +324,7 @@ The following attributes are assigned to this type of data set `value.VN`:
 
 #### Sample Index
 
-If datasets are created for coordinates, a data set on the same level and with the name `sampleindex` must be available. The index number of an entry in `coord.CN`, is entered as the start of the next group. If the data set `sampleindex` has a value of $21$ at index zero, the first $20$ entries from the data set `coord.CN` belong together, the next data group start at with index number $21$. The number of entries in `sampleindex` corresponds to the one in the data set `timestamp`.
+If datasets are created for coordinates, a data set on the same level and with the name `sampleindex` must be available. The index number of an entry in `coord.CN`, is entered as the start of the next group. If the data set `sampleindex` has a value of $21$ at index zero, the first $20$ entries from the data set `coord.CN` belong together, the next data group starts at with index number $21$. The number of entries in `sampleindex` corresponds to the one in the data set `timestamp`.
```

Comment:
not clear:
the next data group starts at ...
or
the next data group starts with ...
(or something else ?)

Reply:
Thanks  for the comment Serge. It should say: "...the next data group starts at index..."